src/index.js

```diff
@@ -25,8 +25,14 @@ var getRev = function (osVersion) {
         return 0;
     }
 };
-var isDevelopmentVersion = function (version) {
-    return /(\.|\+|-)dev/.test(version);
+var getPreReleaseVariant = function (version) {
+    var matches = /-(dev|prod)/.exec(version);
+    return matches && matches[1] || '';
+};
+var isPreReleaseVariant = function (version) { return !!getPreReleaseVariant(version); };
+var getPostReleaseVariant = function (version) {
+    var matches = /([\.|\+](dev|prod))|(\((\w+)\))/.exec(version);
```

Comment:
This will also match prerelease variants (`2.0.0-beta.dev`), which isn't what you want.